docs/manage/data/_index.md

```diff
@@ -30,6 +30,13 @@ Once you have configured data capture and cloud sync, you can <a href="/manage/d
 
 {{< /cards >}}
 
+{{% alert title="Tip" color="tip" %}}
+
+Data management permissions vary between organization owners and operators.
```

Comment:
```suggestion
Data management permissions vary between owners and operators.
```

Changed this to just "owners and operators" since we will have location and robot owners and operators as well.